pkgs/development/tools/boost-build/default.nix

```diff
@@ -39,6 +39,8 @@ stdenv.mkDerivation {
   postPatch = ''
```

Comment:
@wegank What are your thoughts on this?

```suggestion
  postPatch = lib.optionalString stdenv.isDarwin ''
```

Reply:
I prefer to keep patches as general as possible, so I'm not encouraging this.

Reply:
Thanks for the helpful clarification 🙂